### PR TITLE
rpcserver: avoid nested decodescript p2sh addrs

### PR DESCRIPTION
**Pull request by dnldd**
This PR contains the following upstream commits:

- [a6bf1d9](https://github.com/decred/dcrd/commit/a6bf1d9850f857f8971204b62c466c4b0f7be2ac)
    - NOOP
- [a82f67b](https://github.com/decred/dcrd/commit/a82f67b538d50501741a3322238218577fd62669)
    - NOOP
- [e7caccc](https://github.com/decred/dcrd/commit/e7caccc866b2caef61aefc9790a952c94cd04fd5)
    - NOOP
- [403aaf5](https://github.com/decred/dcrd/commit/403aaf5cf31eabb8408740d2fb3df0de9d1fa257)

### CODE REVIEW

File: rpcserverhelp.go [hunk 1]
```diff
@@ -156,8 +156,8 @@ var helpDescsEnUS = map[string]string{
 	"decodescriptresult-asm":       "Disassembly of the script",
 	"decodescriptresult-reqSigs":   "The number of required signatures",
 	"decodescriptresult-type":      "The type of the script (e.g. 'pubkeyhash')",
-	"decodescriptresult-addresses": "The decred addresses associated with this script",
-	"decodescriptresult-p2sh":      "The script hash for use in pay-to-script-hash transactions",
+	"decodescriptresult-addresses": "The bitcoin addresses associated with this script",
```
Comment (jrick) [inline rpcserverhelp.go]: shouldn't this remain decred?

Comment (davecgh) [reply rpcserverhelp.go]: Good catch.  I glossed right over that.

Comment (davecgh) [reply rpcserverhelp.go]: Good catch.  I glossed right over that while checking the merges were indeed NOOPs and everything is already applied.

Comment (dnldd) [reply rpcserverhelp.go]: oops, correcting.

**Pull request merged**